eng/pylintrc

```diff
@@ -3,8 +3,11 @@ ignore-patterns=test_*,conftest,setup
 reports=no
 
 # PYLINT DIRECTORY BLACKLIST.
-ignore=_vendor,_generated,_restclient,samples,examples,test,tests,doc,.tox
-ignore-paths=azure\\mixedreality\\remoterendering\\_api_version.py,azure/mixedreality/remoterendering/_api_version.py
+ignore-paths=
+    azure\\mixedreality\\remoterendering\\_api_version.py,
+    azure/mixedreality/remoterendering/_api_version.py,
+    # Exclude any path that contains the following directory names
+    (?:.*[/\\]|^)(?:_vendor|_generated|_restclient|samples|examples|test|tests|doc|.tox)(?:[/\\]|$)
```

Comment:
@kdestin I seriously doubt that you haven't run this locally, but just confirming that you don't need to escape `/`.

[regex101](https://regex101.com/r/zwHTol/1) definitely looks good to me.